### PR TITLE
Person student && Teacher class

### PR DESCRIPTION
**Pull request by mckent05**
# In this PR, I have successfully completed the following:
- Created a Person class with instance variables
- Created a Student class that inherits the Person class
- Created a Teacher class that inherits the Person class
- Created all required methods for respective classes
- Ensured that I have met all project requirements

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown')
+    super(age, name)
```
Comment (mikethreels) [inline student.rb]: - [ ] in the `Student` class and `Teacher` class `parent_permission = true` should be passed as an argument as well.

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown')
+    super(age, name)
```
Comment (mikethreels) [inline teacher.rb]: - [ ] in the `Student` class and `Teacher` class `parent_permission = true` should be passed as an argument as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Tope, 

Good job creating the `Person` class and including the private method 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @mckent05,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**